src/main/java/org/apache/ibatis/migration/commands/BaseCommand.java

```diff
@@ -106,6 +111,23 @@ public void setPrintStream(PrintStream aPrintStream) {
     printStream = aPrintStream;
   }
 
+  /**
+   * Use this to define template variables
+   * @return combined properties of system/environment with 'sys.' and 'env.' appended respectively
+   */
+  protected Properties getVariables() {
```

Comment:
The templating system for `new` only had `description` available which was really limited. In the case I have in mine I need some environment variables. This falls in line with typical template substitutions. So, in your `new script` you can do this: `insert into ${env.USER}` this was previously not available. Because I put it in `BaseCommand` it is available to all commands now to integrate.